README.md

```diff
@@ -380,6 +380,10 @@ If you cannot find suitable test data on this repository, please contact us on t
     - P53_6samples.cls: a categorical (e.g tumor vs normal) class file format (CLS) example
     - c1.symbols.reduced.gmx: a GMX (Gene MatriX file format) example
 
+  - array_expression:
+    - GSE38751.csv: Sample sheet describing Affy array CEL files
+    - '*.CEL.gz': compressed array CEL files
```

Comment:
This file hasn't been added? Also the tarred folder is missing :)

Reply:
Arghh- sorry. I changed my mind how to handle this and forgot to update the docs- thanks for the catch.

Reply:
No problem!!